plugins/Toolbox/src/Toolbox.py

```diff
@@ -169,6 +169,27 @@ def _onAppInitialized(self) -> None:
             "materials_showcase": QUrl("{base_url}/showcase".format(base_url=self._api_url))
         }
 
+    OLD_PLUGINS = ['3DPrinterOS',
+                   'BarbarianPlugin',
+                   'CuraSolidWorksPlugin',
+                   'MakePrintablePlugin',
+                   'OctoPrintPlugin',
+                   'OrientationPlugin',
+                   'ZOffsetPlugin',
+                   'cura-siemensnx-plugin']
+
+    # check for plugins that were installed with the old plugin-browser
+    def _isOldPlugin(self, plugin_id) -> bool:
```

Comment:
Missing typing for `plugin_id` (`plugin_id: str`)

Reply:
fixed